backend/models/event_details.py

```diff
@@ -0,0 +1,13 @@
+from backend.models.event import Event
+from backend.models.organization import Organization
+    
+class EventDetails(Event):
+    """
+    Pydantic model to represent an `Event`, including back-populated
+    relationship fields.
+
+    This model is based on the `EventEntity` model, which defines the shape
+    of the `Event` database in the PostgreSQL database.
+    """
+    
+    organization: Organization | None = None
```

Comment:
Should this actually be `| None`?  Is there a case when an event has no organization affinity?

Reply:
(And if there is... how do we add and edit them :)